Server/src/genericInterfaces/AuthenticationInterfaces.ts

```diff
@@ -18,6 +18,12 @@ export interface IJwtParams {
     firstName: string
 }
 
+export interface IUserDetailUpdater {
```

Comment:
Its confusing with the other interface below, maybe we can call this IUpdateUserDetail, and the one below IFetchUserDetail

Reply:
fixed in the latest commit